src/config.js

```diff
@@ -0,0 +1 @@
+export const LOGIN_URL = 'http://10.58.7.139:8000/users/kakao/callback';
```

Comment:
```suggestion
export const BASE_URL = 'http://10.58.7.139:8000/';
export const API = {
  KAKAO_LOGIN: `${BASE_URL}users/kakao/callback`,
  CART: `${BASE_URL}cart`,
};

```

예를들면 endpoint별로 요렇게 관리할 수 있습니다 :-)

Reply:
객체로 관리할 수 있다는 것을 처음 알았습니다.감사합니다 수정 하였습니다!